charts/openyurt/templates/yurt-manager-auto-generated.yaml

```diff
@@ -878,162 +1528,3 @@ rules:
   - get
   - patch
   - update
----
```

Comment:
why do you remove wehbook configurations?

Reply:
The webhooks are not deleted, the codes are regenerated.